buildcatrust/certstore_output.py

```diff
@@ -29,6 +30,34 @@ def output(self, cert: Optional[types.Certificate], trust: types.Trust) -> None:
                 file=self.fp,
             )
 
+
+class StandardCertStoreOutput:
+    """
+    PEM standard abiding output when the OpenSSL specific format
+    with trust rules is incompatible with your software.
+    """
+
+    def __init__(self, fp: TextIO):
+        self.fp = fp
+
+    def output(self, cert: Optional[types.Certificate], trust: types.Trust) -> None:
+        if not cert:
+            return
+
+        print(cert.label, file=self.fp)
+
+        # Output the "plain" version for applications expecting just plain CERTIFICATE entries.
+        # We only do this if the cert is affirmatively trusted for being a CA for server-side auth.
+        print(cert.as_pem().encode(), file=self.fp)
```

Comment:
You don't check the trust bit here.

Reply:
Alright, I was not sure that distrusted should still be outputted or not